addon-test-support/@ember/test-helpers/setup-context.ts

```diff
@@ -236,6 +259,79 @@ export function getDeprecationsDuringCallback(
   return getDeprecationsDuringCallbackForContext(context, callback);
 }
 
+/**
+ * Returns warnings which have occured so far for a the current test context
+ *
+ * @public
+ * @returns {Array<Warning>} An array of deprecation messages
```

Comment:
```suggestion
 * @returns {Array<Warning>} An array of warnings
```

1) it isn't deprecatiosn
2) it is an object with `{ message: 'whatever', options: {} }` (not just messages)